docs/source/algorithms/ConvertQtoHKLMDHisto-v1.rst

```diff
@@ -0,0 +1,19 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides a way to convert from a `MDEventWorkspace` in QSample units to a `MDHistoWorkspace` in HKL
+units. By default, the UB matrix from the `InputWorkspace` is used unless `FindUBFromPeaks` is true. In this case, the
+UB matrix from the specified `PeaksWorkspace` will be used. An optional transformation can be applied to the peak
+UB matrix to better align peaks in HKL space (see :ref:`TransformHKL <algm-TransformHKL>`).
+
```

Comment:
Can you add a note that "this doesn't correctly take into account the normalization of the output, it's simply binning the data in HKL. Look at MDNorm (it using TOF) to correctly take into account the normalization when converting to HKL"

Reply:
You should add some formula, or link to the lattice concept as well